webapp/pages/profile/_id/_slug.vue

```diff
@@ -430,6 +430,14 @@ export default {
         },
         fetchPolicy: 'cache-and-network'
       })
+    },
+    userName(maxLength) {
+      // Return Anonymous if no Username is given
+      if (!this.user.name) {
+        return this.$t('profile.userAnonym')
+      }
+      // Return full Username or truncated Username
+      return maxLength ? this.user.name.substring(0, maxLength) : this.user.name
```

Comment:
Actually I like `truncate` more. Does it add ellipsis ... when you have a long user name?

Reply:
![image](https://user-images.githubusercontent.com/1238238/56419111-755c1c80-6299-11e9-95cf-f8dd5a8116cd.png)

using truncate now